ACQ-ID New Floppy and Iomega tranfer pages (#92)
NYPL/digarch: transfers/floppy-disk-imaging.md

parent: Transfers
---

# Floppy Disks

This page describes instructions for imaging floppy disks using the KryoFlux microcontroller and software. For more detailed installation and use information see the [Kryoflux documentation](https://nypl.github.io/digarch/tools/kryoflux) in our Tools section.

### Before Imaging

> * Before any media can be imaged **it first must have an object record in SPEC**.

Reviewer: ```suggestion
{: .important }
Before a media object can be imaged it first must be recorded SPEC.
```

> * Verify a tracking card exists on Trello for the associated acquisition being imaged, create a card if it does not already exist.

Reviewer: ```suggestion
* Verify a tracking card exists on Trello for the associated acquisition being imaged  
    * Create a card if it does not already exist  

```

| -- |  
| Y |  

> ### Package Images with package_images.py

Reviewer: Align Package Images instructions for Floppy and Iomega

The python script package_images.py identifies digital carrier images in an input directory, creates an archival information package, and moves image files into the package. This section describes packaging with package_images.py for floppy and Iomega zip disks. For detailed information on installation and use see the [package_image.py](https://nypl.github.io/digarch/tools/working-scripts.html#package_imagespy) listing in the Working Scripts section of our documentation.

Packaging images from floppy disks requires five arguments:
* ```--acqid```: Acquisiton id following the syntax ACQ-####.
* ```--images_folder```: Path to directory holding images to be packaged, typically this will be /Working on DigArchDiskStation.
* ```--logs```: Path to directory holding transfer logs.

> * ```--steams_folder```: Path to directory holding stream files.

Reviewer: ```suggestion
* ```--streams_folder```: Path to directory holding stream files.
```suggestion
```

* ```--dest```: Path to destination directory for package, typically this will be /Transfers on DigArchDiskStation or on Isilon in the case of large acqusiitons.

Use package_images.py with the followings steps:
* Open Terminal
* Enter the following command with the appropriate paths: ```python3 package_images.py –-acqid (M#### or ACQ_####) -–source path/to/images -–dest path/to/diskImages```

> * Navigate to destination directory to verify packages have been created and image files have been moved

Reviewer: ```suggestion
* Navigate to destination directory to verify packages were created and image files were moved
```
